runtime/jcl/common/java_lang_ref_Reference.cpp

```diff
@@ -81,11 +77,7 @@ Java_java_lang_ref_Reference_refersTo(JNIEnv *env, jobject reference, jobject ta
 		j9object_t j9target = (NULL != target) ? J9_JNI_UNWRAP_REFERENCE(target) : NULL;
 		j9object_t referent = NULL;
 
-		if (J9_GC_POLICY_METRONOME == ((OMR_VM *)vm->omrVM)->gcPolicy) {
-			referent = vm->memoryManagerFunctions->j9gc_objaccess_referenceGet(currentThread, j9reference);
-		} else {
-			referent = J9VMJAVALANGREFREFERENCE_REFERENT_VM(vm, j9reference);
-		}
+		referent = vm->memoryManagerFunctions->j9gc_objaccess_referenceGet(currentThread, j9reference);
```

Comment:
you can merge this into a single line with declaration

same pattern later in the code, too

Reply:
Could you please clarify, I'm not too sure how to address this

Reply:
Do you mean that I should move this into the `if` statement on the following line?

Reply:
```C++
if ((referent = vm->memoryManagerFunctions->j9gc_objaccess_referenceGet(currentThread, j9reference)) == j9target) {
			result = JNI_TRUE;
		}
```

Reply:
I meant just this 

```
j9object_t referent = vm->memoryManagerFunctions->j9gc_objaccess_referenceGet(currentThread, j9reference);
```